exporter/dynatraceexporter/metrics_exporter.go

```diff
@@ -164,12 +166,16 @@ func (e *exporter) send(ctx context.Context, message string) (int, error) {
 		e.logger.Debug(fmt.Sprintf("Accepted %d lines", responseBody.Ok))
 		e.logger.Error(fmt.Sprintf("Rejected %d lines", responseBody.Invalid))
 
-		if responseBody.Error != "" {
-			e.logger.Error(fmt.Sprintf("Error from Dynatrace: %s", responseBody.Error))
+		if responseBody.Error.Message != "" {
+			e.logger.Error(fmt.Sprintf("Error from Dynatrace: %s", responseBody.Error.Message))
 		}
 
-		return responseBody.Invalid, nil
+		for _, line := range responseBody.Error.InvalidLines {
+			// Enabled debug logging to see which lines were dropped
+			e.logger.Debug(fmt.Sprintf("line %3d %s %s", line.Line, line.Error, lines[line.Line]))
```

Comment:
`line.Line` as returned by the server should not be out of bounds but we should still check it nevertheless